mailhandler/spameater

```diff
@@ -421,13 +425,15 @@ $config->debug('actually not... the account is disabled.');
       $for =~ /(.*)\@/;
       # if there was no special syntax (i.e., this is just a plain address)
       if ($musername eq $1) {
-        &sendMail($config,$useXHeader,$RealEmail,\$msg,'');  # just deliver the message
-        next;                      # and skip the rest of the loop 
+        &sendMail($config,$useXHeader,\@rcpts,\$msg,'');  # just deliver the message
```

Comment:
If I'm reading this correctly, `@rcpts` could be either `$subfrom`, the adminf email, or unassigned by this line? Is that preferrable to it staying the `$musername` it would have been before?